webgui/scripts/codecompass/view/header.js

```diff
@@ -54,6 +54,14 @@ function (cookie, topic, declare, DropDownMenu, DropDownButton, Select,
         }
       }));
 
+      menu.addChild(new MenuItem({
+        label     : 'New Web GUI',
+        onClick   : function () {
+          window.open(
+            `${window.location.origin}/new`);
```

Comment:
```suggestion
            `${window.location.pathname}/new`);
```

Reply:
So it is compatible with URLs where the CodeCompass GUI not served under the root folder.